### PR TITLE
Rename customer to client in User Guide

### PR DESCRIPTION
**Pull request by itsme-zeix**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -403,36 +403,36 @@ For detailed explanations of each flag and acceptable arguments, refer to Sectio
 - **On Success:**
     - Message:
       ```
-      Customer <INDEX> has been updated successfully.
+      client <INDEX> has been updated successfully.
```
Comment (FionaQY) [inline docs/UserGuide.md:406]: The message for DeleteCommand has a capitalised 'Client' so I think this should be capitalised as well

**Review by FionaQY — APPROVED** [review]
LGTM. Just a nix

File: docs/UserGuide.md (hunk 1 above)
Comment (iamdiluxedbutcooler) [reply docs/UserGuide.md:406]: good catch!

**Review by iamdiluxedbutcooler — APPROVED** [review]
LGTM! Can double check whether the link between topics work fine on markdown after we merge.

File: docs/UserGuide.md (hunk 1 above)
Comment (itsme-zeix) [reply docs/UserGuide.md:406]: Nice catch!

**Pull request merged**